README.markdown

```diff
@@ -3641,160 +3641,282 @@ The pool is global at Nginx worker level. And it is used to reuse Lua VMs betwee
-* [ngx.shared.DICT.lpop](#ngxshareddictlpop)
-* [ngx.shared.DICT.rpop](#ngxshareddictrpop)
-* [ngx.shared.DICT.llen](#ngxshareddictllen)
-* [ngx.shared.DICT.ttl](#ngxshareddictttl)
-* [ngx.shared.DICT.expire](#ngxshareddictexpire)
-* [ngx.shared.DICT.flush_all](#ngxshareddictflush_all)
-* [ngx.shared.DICT.flush_expired](#ngxshareddictflush_expired)
-* [ngx.shared.DICT.get_keys](#ngxshareddictget_keys)
-* [ngx.shared.DICT.capacity](#ngxshareddictcapacity)
-* [ngx.shared.DICT.free_space](#ngxshareddictfree_space)
-* [ngx.socket.udp](#ngxsocketudp)
-* [udpsock:bind](#udpsockbind)
-* [udpsock:setpeername](#udpsocksetpeername)
-* [udpsock:send](#udpsocksend)
-* [udpsock:receive](#udpsockreceive)
-* [udpsock:close](#udpsockclose)
-* [udpsock:settimeout](#udpsocksettimeout)
-* [ngx.socket.stream](#ngxsocketstream)
-* [ngx.socket.tcp](#ngxsockettcp)
-* [tcpsock:bind](#tcpsockbind)
-* [tcpsock:connect](#tcpsockconnect)
-* [tcpsock:setclientcert](#tcpsocksetclientcert)
-* [tcpsock:sslhandshake](#tcpsocksslhandshake)
-* [tcpsock:send](#tcpsocksend)
-* [tcpsock:receive](#tcpsockreceive)
-* [tcpsock:receiveany](#tcpsockreceiveany)
-* [tcpsock:receiveuntil](#tcpsockreceiveuntil)
-* [tcpsock:close](#tcpsockclose)
-* [tcpsock:settimeout](#tcpsocksettimeout)
-* [tcpsock:settimeouts](#tcpsocksettimeouts)
-* [tcpsock:setoption](#tcpsocksetoption)
-* [tcpsock:setkeepalive](#tcpsocksetkeepalive)
-* [tcpsock:getreusedtimes](#tcpsockgetreusedtimes)
-* [ngx.socket.connect](#ngxsocketconnect)
-* [ngx.get_phase](#ngxget_phase)
-* [ngx.thread.spawn](#ngxthreadspawn)
-* [ngx.thread.wait](#ngxthreadwait)
-* [ngx.thread.kill](#ngxthreadkill)
-* [ngx.on_abort](#ngxon_abort)
-* [ngx.timer.at](#ngxtimerat)
-* [ngx.timer.every](#ngxtimerevery)
-* [ngx.timer.running_count](#ngxtimerrunning_count)
-* [ngx.timer.pending_count](#ngxtimerpending_count)
-* [ngx.config.subsystem](#ngxconfigsubsystem)
-* [ngx.config.debug](#ngxconfigdebug)
-* [ngx.config.prefix](#ngxconfigprefix)
-* [ngx.config.nginx_version](#ngxconfignginx_version)
-* [ngx.config.nginx_configure](#ngxconfignginx_configure)
-* [ngx.config.ngx_lua_version](#ngxconfigngx_lua_version)
-* [ngx.worker.exiting](#ngxworkerexiting)
-* [ngx.worker.pid](#ngxworkerpid)
-* [ngx.worker.pids](#ngxworkerpids)
-* [ngx.worker.count](#ngxworkercount)
-* [ngx.worker.id](#ngxworkerid)
-* [ngx.semaphore](#ngxsemaphore)
-* [ngx.balancer](#ngxbalancer)
-* [ngx.ssl](#ngxssl)
-* [ngx.ocsp](#ngxocsp)
-* [ndk.set_var.DIRECTIVE](#ndkset_vardirective)
-* [coroutine.create](#coroutinecreate)
-* [coroutine.resume](#coroutineresume)
-* [coroutine.yield](#coroutineyield)
-* [coroutine.wrap](#coroutinewrap)
-* [coroutine.running](#coroutinerunning)
-* [coroutine.status](#coroutinestatus)
-* [ngx.run_worker_thread](#ngxrun_worker_thread)
+- [Name](#name)
+- [Table of Contents](#table-of-contents)
+- [Status](#status)
+- [Version](#version)
+- [Videos](#videos)
+- [Synopsis](#synopsis)
+- [Description](#description)
+- [Typical Uses](#typical-uses)
```

Comment:
please don't change the format of the doc.

Reply:
ok,i just know this ,sorry